packages/create-sheriff-config/package.json

```diff
@@ -26,7 +25,7 @@
     "yargs": "^17.7.2"
   },
   "bin": {
-    "create-sheriff-config": "./dist/index.js"
+    "create-sheriff-config": "dist/index.js"
```

Comment:
why was this needed?

Reply:
It's not, technically...
I ran [`npm pkg fix`](https://docs.npmjs.com/cli/v10/commands/npm-pkg#:~:text=delete%20scripts.build-,npm%20pkg%20fix,-Auto%20corrects%20common) to normalize the `package.json`s. It makes what goes in node_modules the same, otherwise it's different. It's so that the server serves the same thing, so that the (p)npm client(s) can be more forgiving while still using a single format in-memory.
Canonically, `bin`s apparently oughtn't have leading `./`, but (p)npm fixes it while publishing. This just commits those changes.